tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = py27,py34,py35,py36,pypy
+
+[testenv]
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+deps =
+    --no-binary
+     zc.recipe.egg
```

Comment:
You're one of the [lucky 10000](https://xkcd.com/1053/) today!

The GNU getopt_long(3), which introduced long options, allows both `--option value` and `--option=value`.  Python's optparse/argparse also support both spellings (and this is even [documented](https://docs.python.org/3/library/argparse.html#option-value-syntax) deep in the reference manual that nobody reads for fun).

I've been used to this for so long that I'm always surprised when I discover a program (*cough* gnome-session-inhibit *cough*) that doesn't accept --foo=bar.